demo-app/src/component.js

```diff
@@ -14,9 +14,10 @@ window.onhashchange = function() {
 let iframeClient = new Client();
 iframeClient.start();
 
-iframeClient.onPubsub(publication => {
+iframeClient.on('publish', publication => {
   console.log("Got Publish event:", publication);
 });
+
 iframeClient.subscribe("host.topic");
```

Comment:
This is one of my bigger open questions for API design. 

With this approach, we would have to still subscribe to publish topics.  But it provides a common API for any event that we wanted to publish.

Alternatively we could say that the on will only emit Publish events.
Thus our api would be `.on('host.topic', handler)`

Now that I write this, I think that might be better...

Reply:
Updated the API to `.on('host.topic', handler)` style API.